src/core/InstrumentFunctions.cpp

```diff
@@ -408,7 +408,37 @@ void InstrumentFunctionArpeggio::processNote( NotePlayHandle * _n )
 			continue;
 		}
 
-		const int dir = m_arpDirectionModel.value();
+		// Skip notes randomly
+		if( m_arpSkipModel.value() )
+		{
+
+			if( 101 * ( (float) rand() / (float) RAND_MAX ) < m_arpSkipModel.value() )
```

Comment:
Why `101 * r / RM < skip` instead of `100 * r / (RM + 1) < skip`? 100% skip rate should skip always.